install/operator/pkg/generator/assets/addons/todo/04-todo-example.yml.tmpl

```diff
@@ -75,9 +75,8 @@
     strategy:
       sourceStrategy:
         from:
-          kind: ImageStreamTag
-          name: 'php:7.0'
-          namespace: openshift
+        kind: DockerImage
+        name: 'docker.io/centos/php-71-centos7'
```

Comment:
For product, this should probably be a different image (probably `rhscl/php-71-rhel7` though not 100 % sure). For images that need to be overwritten, we generally put those into `config.yaml`, I think that's what we need to do here as well, so that productization can override it. 

cc @orpiske